templates/batchConversion.tpl

```diff
@@ -0,0 +1,52 @@
+{**
+ * templates/batchConversion.tpl
```

Comment:
This template generally needs review for escaping. In most cases, passing output through the `escape` Smarty modifier is enough.

Reply:
https://github.com/kaschioudi/ojs3-markup/commit/9f1f27b21a31ec1408a91a399782ce2c97b6f38e